src/cfnlint/core.py

```diff
@@ -157,10 +157,26 @@ def get_args_filenames(cli_args):
 def get_template_rules(filename, args):
     """ Get Template Configuration items and set them as default values"""
 
-    (template, matches) = cfnlint.decode.decode(filename, args.ignore_bad_template)
-
-    if matches:
-        return(template, [], matches)
+    ignore_bad_template = False
+    if args.ignore_bad_template:
+        ignore_bad_template = True
+    else:
+        # There is no collection at this point so we need to handle this
+        # check directly
+        if not ParseError().is_enabled(
```

Comment:
this should now handle determining if the ParseError is enabled just like a normal rule.